src/pvecontrol/actions/cluster.py

```diff
@@ -33,21 +35,54 @@ def _get_disk_output():
         d_percent = metrics["disk"]["percent"]
         return f"{d_usage}/{d_total}({d_percent:.1f}%)"
 
-    output = f"""
-  Status: {status}
-  VMs: {vms - templates}
-  Templates: {templates}
-  Metrics:
-    CPU: {_get_cpu_output()}
-    Memory: {_get_memory_output()}
-    Disk: {_get_disk_output()}
-  Nodes:
-    Offline: {len([node for node in proxmox.nodes if node.status == NodeStatus.OFFLINE])}
-    Online: {len([node for node in proxmox.nodes if node.status == NodeStatus.ONLINE])}
-    Unknown: {len([node for node in proxmox.nodes if node.status == NodeStatus.UNKNOWN])}
-  """
-
-    print(output)
+    if _args.output == OutputFormats.TEXT:
+        output = f"""\
+            Status: {status}
+            VMs: {vms - templates}
+            Templates: {templates}
+            Metrics:
+                CPU: {_get_cpu_output()}
+                Memory: {_get_memory_output()}
+                Disk: {_get_disk_output()}
+            Nodes:
+                Offline: {len([node for node in proxmox.nodes if node.status == NodeStatus.OFFLINE])}
+                Online: {len([node for node in proxmox.nodes if node.status == NodeStatus.ONLINE])}
+                Unknown: {len([node for node in proxmox.nodes if node.status == NodeStatus.UNKNOWN])}
+            """
```

Comment:
In a sense we can argue that this provides a "better output", but in the other hand I was quite fine with the previous output. I feel like it was intentional to prepend every lines with a double space and start the output with a `\n` to clearly delimitate the output from previous logs (`INFO:root:Proxmox cluster: XXX` for instance).

Any thoughts about it @lcaflc?